notebooks/00_quick_start/dkn_synthetic.ipynb

```diff
@@ -22,13 +22,20 @@
     "- DKN uses an attention module to dynamically calculate a user's aggregated historical representaition.\n",
```

Comment:
<p>(0) Replace "needed files" with "DKN takes several files as input as follows":</p><p>(1) training / validation / test files: each line in these files represents one instance, the format is like: </p><p>  <code>[label] [userid] [CandidateNews] [impressionid]</code></p><p> e.g.,  [Please give one example here],   the same goes for the other illustrations.</p><p>And state what is the separator in each line.</p><p>(2)How about the word embedding and entity embedding? Why not added to here yet</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
done